crawl-ref/docs/develop/species_creation.md

```diff
@@ -29,7 +29,7 @@ You can view existing files to get a sense of the format, it's quite straightfor
 | int | `int` | **Yes** | Starting intelligence. |
 | dex | `int` | **Yes** | Starting dexterity. |
 | levelup_stat_frequency | `int` | **Yes** | How frequently (in XLs) the species' periodic stat-up occurs. You can specify `28` to ensure the species never gains stats on levelup. Note that all species gain a user-selectable stat every three levels. |
-| levelup_stats | `array of strings` | No | Which stats the species' periodic stat-up can randomly select from. Defaults to `[str, dex, int]`. |
+| levelup_stats | `array of strings` or `str` | No | Which stats the species' periodic stat-up can randomly select from. Defaults to `[str, dex, int]`. Set to `none` for cases where levelup stats happen outside of the usual system. |
```

Comment:
We should ensure that if `levelup_stats` is `none`, `levelup_stat_frequency` is `28`. ...Right? I imagine the game will crash otherwise.